File: Scripts/src/UpdateManager.cs
using UnityEngine;

class UpdateManager
{
    private static UpdateManager _instance;

    public static UpdateManager Instance
    {
        get
        {
            if (_instance == null)
            {
                _instance = new UpdateManager();
            }
            return _instance;
        }
    }

    public void Start()
    {
        Debug.Log("UpdateManager Start");
        UIManager.Instance.ShowPanel("UpdatePanel");
        HttpManager.Instance.StartHttpGet("http://www.baidu.com", content =>
        {
            Debug.Log(content);
        });
    }

    public void OnDestroy()
    {
        Debug.Log("UpdateManager OnDestroy");
    }
}

File: JavaScript/UpdateManager.js
var UpdateManager = {
    fullname: "UpdateManager",
    definition: {
        Start: function (){
            print("UpdateManager Start");
			UIManager.get_Instance().ShowPanel("UpdatePanel");
            HttpManager.get_Instance().StartHttpGet("http://www.baidu.com", function(str) {
				SceneManagerEx.get_Instance().ShowScene("Login");
				UIManager.get_Instance().ClosePanel("UpdatePanel");
            });
        }
    }
};
JsTypes.push(UpdateManager);